requirements.txt

```diff
@@ -41,3 +42,10 @@ django-ptvsd-debug==1.0.3
 django-lti-auth==1.0.0
 django-webpack-loader==0.6.0
 django-csp==3.5
+
+# TypingDict and Protocol are not available until Python 3.8 (PEP 589)
+typing-extensions==3.7.4.1
+
+# For running the type checking in the container
+mypy==0.761
```

Comment:
Why do we need to exactly equal a particular version for `mypy`, `django-hint` and `typing-extensions`?

Reply:
I think for mypy we'd actually want to test to make sure it passed with a higher version. We probably don't need to lock the others down as hard.

Reply:
Looking at the release schedules, I think using `==` for `mypy` and `django_hint` is fine. We could probably use a `>=` and `<=` pattern for `typing-extensions` (something like `>=3.7.4.1,<=3.7.4.9`).

Reply:
Ok, I'll use that